versioned_docs/version-1.15/using-dev-envs.mdx

```diff
@@ -63,7 +63,7 @@ When it's done building the images, it will start deploying your application to
   - https://movies-with-compose-cindylopez.example.okteto.com/
 ```
 
-These endpoints are also shown in the Okteto dashboard.
+The deployed endpoint is also shown in the Okteto dashboard.
```

Comment:
There's only one endpoint created so I find that using the plural is a bit confusing.